### PR TITLE
Made a Art section of HomePage

### PR DESCRIPTION
**Pull request by meena730**
## Description

-   [Ticket Link](#) (if any)
-   Provide a small description ( 2 - 3 lines) of the Pull Request

## UI screenshot or Video

-   Please include a screenshot or video to help reviewers understand the specific part of the application being modified. Multiple screenshots are appreciated

---

### General:

-   [x] **PR Size :** Confirming PR size < 20 files or pre-approved if > 20 files.
-   [x] **Self Review :** Confirming I have self-reviewed my PR
-   [x] **Peers Review :** - [ ] @Dev 1
-   [x] Confirming I have added all sensitive information-related should be inside .env files

### Client:

-   [x] **Console :** Confirming no console errors in the UI for this update.

-   [x] **Import order :** Confirming correct import order.

```
# External Import
import React, { useState, useEffect } from 'react';
import { useParams } from 'react-router-dom';
import { useDispatch, useSelector, shallowEqual } from 'react-redux';

# Internal Import
import { } from '../../Api/ApiRoutes';
import { } from '../../constants/AppConstant';
```

-   [x] **Re-renders :** Confirming minimal re-renders for new React components.

-   [ ] **Network calls :** Confirming efficient network/API calls, minimized as needed.

-   [ ] **Util function comments :** Confirming comprehensive comments for utility functions.

-   [x] **Image optimization :** Confirming I have checked that all images are optimized. If not, please use https://tinyjpg.com/ or https://vecta.io/nano.

-   [x] **Dead code :** Confirming I have removed all comments code which is of no use OR written why I have kept the commented code.

### Confirming different viewport testing of the entire flow:

-   [x] Tested on 280px viewport
-   [x] Tested on 320px viewport
-   [x] Tested on 375px viewport
-   [x] Tested on 425px viewport
-   [x] Tested on 768px viewport
-   [x] Tested on 1024px viewport
-   [x] Tested on 1440px viewport
-   [x] Tested on 2560px viewport

### End-to-end testing:

-   [x] **Happy flow :** Successfully tested end-to-end happy flow, all working as expected.
-   [x] **Error flow :** Tested UI flow with error scenarios, gracefully handled errors on UI
-   [x] **Common component changed :** Tested change in common component across views to prevent breaking elsewhere.

### CODE REVIEW

File: src/pages/Home/Art.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import art from '../../assets/HomeImg/Art.png';
+import gridline from '../../assets/HomeImg/gridline-bg.png';
+import Scrollservice from './Scroll-service';
+import NewsCard from './NewsCard';
+
+function Art() {
```
Comment (Kuldeep-primathon) [inline src/pages/Home/Art.jsx]: Component name should be meaningful eg: Art-> ArtSection

File: src/pages/Home/Art.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import art from '../../assets/HomeImg/Art.png';
```
Comment (Kuldeep-primathon) [inline src/pages/Home/Art.jsx]: Put image directly in assets folder or create images folder in assets and put all images inside it and image name should be in kebab case

File: src/pages/Home/Art.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import NewsCard from './NewsCard';
+
+function Art() {
+    return (
+        <>
+            <div className="px-[20px] md:px-[56px] lg:px-[82px] xl:px-[100px] 2xl:px-[120px] 3xl:px[182px] pt-[48px]">
+                <section>
+                    <div className="pl-[18px]">
+                        <div className="hidden lg:block lg:relative opacity-1 lg:h-0 lg:top-[-6.668rem] lg:left-[-5.2rem] xl:left-[-7.7rem] 2xl:left-0">
+                            <img src={gridline} alt="Grid" className="md:w-full md:h-96" />
```
Comment (Kuldeep-primathon) [inline src/pages/Home/Art.jsx]: Image name and alt attribute should be more descriptive

File: src/pages/Home/Scroll-service.jsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        <section>
+            <div className="w-full flex justify-center items-center pt-5 pb-12 sm:pb-5 z-10">
+                <NavLink
+                    to="/"
+                    className="flex items-center justify-center cursor-pointer gap-8 xs:gap-14 2xl:gap-16 py-1 xl:py-2 pl-5 xl:pl-12 2xl:pl-16 bg-[#F4F8FF] rounded-full hover:bg-gradient-to-r hover:from-white hover:to-pink-100">
+                    <h1 className="text-blue-500 text-xs sm:text-sm md:text-base xl:text-xl font-semibold hover:scale-110 transition-all duration-300">
+                        ALL OUR SERVICES
+                    </h1>
+                    <div className="flex items-center justify-center rounded-full p-3.5 sm:p-4 xl:p-6 mr-1 xs:mr-1.5 xl:mr-2 bg-blue-700 hover:bg-gradient-to-r  hover:from-blue-500 hover:from-10% hover:to-pink-600 transition-all duration-300">
+                        <img src={Arrow} alt="" className="w-3 h-3 sm:w-4 sm:h-4 hover:scale-100" />
```
Comment (Kuldeep-primathon) [inline src/pages/Home/Scroll-service.jsx]: alt should not be empty, Please check all files alt prop

**Comment by Kuldeep-primathon** [comment]
Please follow all the naming convention for (BranchName,Commit ,Images etc) before raising PR

File: src/pages/Home/Art.jsx (hunk 3 above)
Comment (meena730) [reply src/pages/Home/Art.jsx:13]: Done

File: src/pages/Home/Scroll-service.jsx (hunk 4 above)
Comment (meena730) [reply src/pages/Home/Scroll-service.jsx:15]: Noted and resolved

**Pull request closed without merging**